### PR TITLE
feat(bom): use runtime bom

### PR DESCRIPTION
**Pull request by uguy**

### CODE REVIEW

File: src/main/resources/archetype-resources/groovy-pom.xml [hunk 1]
```diff
@@ -44,20 +45,35 @@
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <groovy-maven-plugin.version>2.1.1</groovy-maven-plugin.version>
         <groovy-eclipse-compiler.version>3.7.0</groovy-eclipse-compiler.version>
-        <groovy-eclipse-batch.version>#if( $bonitaVersion.matches('7.(1[3-9]|[2-9][0-9]).*') || $bonitaVersion.matches('[8-99].[0-9]+.*') )
+        <groovy-eclipse-batch.version>#if( $after713 )
 3.0.7-03#else
```
Comment (rbioteau) [inline src/main/resources/archetype-resources/groovy-pom.xml:49]: 3.0.8-01

File: src/main/resources/archetype-resources/pom.xml [hunk 2]
```diff
@@ -42,13 +43,28 @@
         <groovy.version>3.0.7</groovy.version>
```
Comment (rbioteau) [inline src/main/resources/archetype-resources/pom.xml:43]: to be removed

File: src/test/resources/projects/testActorfilterJava11Project/reference/pom.xml [hunk 3]
```diff
@@ -40,13 +40,24 @@
         <groovy.version>3.0.7</groovy.version>
```
Comment (rbioteau) [inline src/test/resources/projects/testActorfilterJava11Project/reference/pom.xml:40]: to be removed

**Review by rbioteau — CHANGES_REQUESTED** [review]
Some little oversight

File: src/main/resources/archetype-resources/pom.xml (hunk 2 above)
Comment (uguy) [reply src/main/resources/archetype-resources/pom.xml]: removed

File: src/test/resources/projects/testActorfilterJava11Project/reference/pom.xml (hunk 3 above)
Comment (uguy) [reply src/test/resources/projects/testActorfilterJava11Project/reference/pom.xml]: removed

File: src/main/resources/archetype-resources/groovy-pom.xml (hunk 1 above)
Comment (uguy) [reply src/main/resources/archetype-resources/groovy-pom.xml]: changed groovy-eclipse-batch version to 3.0.8-01

**Review by rbioteau — APPROVED** [review]

**Pull request merged**